spec/models/category_spec.rb

```diff
@@ -3,4 +3,27 @@
 describe Category do
   it { should have_many (:videos)}
 
+describe "#recent_videos" do
+
+  it "returns all videos if less than 6 exist, in descending order" do
+    documentary = Category.create(name: "documentary")
+    black_fish = Video.create(title: "black fish", description: "awesome", category: documentary, created_at: 1.day.ago)
+    supermensch = Video.create(title: "supermensch", description: "really awesome", category: documentary)
+    expect(documentary.recent_videos).to eq([supermensch, black_fish])
+  end
+
+  it "returns 6 videos in descending order if more than 6 exist" do
+    documentary = Category.create(name: "documentary")
+    7.times { Video.create(title: "supermensch", description: "really awesome", category: documentary) }
+    expect(documentary.recent_videos.count).to eq(6)
```

Comment:
This does not check the order, as the description says it does. Make sure your spec implementations match the descriptions.